src/Orleans/Telemetry/TraceParserUtils.cs

```diff
@@ -70,16 +70,15 @@ public static string PrintProperties(string message, IDictionary<string, string>
                 errorCode);
         }
 
-        /// <summary>
-        /// The method to call during logging to format the log info into a string ready for output.
-        /// </summary>
```

Comment:
When the text is short, it does make sense to avoid multiple lines. This way we reduce the unnecessary comment lines to code lines ratio.
This is not mandatory though, just a nice to have.